docs/content/tools/xdebug.md

```diff
@@ -205,8 +212,6 @@ version: "2.1"
 services:
   cli:
     environment:
-      - XDEBUG_ENABLED=1
```

Comment:
Hey, so why is this removed here. I see the line above says `Ensure that your docksal.yml or docksal-local.yml has been updated to include `XDEBUG_ENABLED=1` in the CLI` but here you removed these 2 lines, why so?

Reply:
this is controlled by the `XDEBUG_ENABLED` variable and is redundant in my opinion.